pkg/pull/signature.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 var (
-	ErrSignatureInvalid = errors.New("license signature is invalid")
-	ErrSignatureMissing = errors.New("license signature is missing")
+	ErrSignatureInvalid = errors.New("signature is invalid")
```

Comment:
Is this sufficient to debug? Should we have license and bundle signature messages as separate strings to make diagnosing problems easier?

Reply:
yeah, stack trace string will be different